src/hooks/useEventEmitter.ts

```diff
@@ -87,7 +87,7 @@ export function useEventEmitterState<T>(
     eventName: string | symbol,
     fn: Mapper<T>,
 ): T {
-    const [value, setValue] = useState<T>(fn());
+    const [value, setValue] = useState<T>(fn);
```

Comment:
Any reason for this change? I would imagine it would gain the same result?

Reply:
Since useState can accept a function as an initializer this doesn't change the behavior, it just prevents `fn` from being needlessly invoked on every render. This also fixes a subtle misbehavior, in that if you passed `() => () => true` as the `fn` you'd initially get `true` as the value, rather than `() => true`, due to the function getting doubly invoked.